darwin/Platform.c

```diff
@@ -18,7 +18,11 @@ in the source distribution for its full text.
 #include <net/if_types.h>
 #include <net/route.h>
 #include <sys/socket.h>
+#ifdef HAVE_SYS__TYPES__MACH_PORT_T_H
 #include <sys/_types/_mach_port_t.h>
+#else
+#include <mach/mach_port.h>
```

Comment:
1. Conditional include lines should be moved to below the mandatory include lines.
2. When I search the code on the web, the type `mach_port_t` seems to be defined in `<mach/port.h>`. I want to make sure you did not confuse `<mach/mach_port.h>` with `<mach/port.h>`.

Reply:
Let me verify and fix if needed.

Reply:
@Explorer09 Ok, so `mach_port_t` is in `mach/port.h`, however `mach/mach_port.h` [includes](https://github.com/alexey-lysiuk/macos-sdk/blob/a79de83d781f7fb76d329aee456bab9f72f935d1/MacOSX10.6.sdk/usr/include/mach/mach_port.h#L14) the former, so it works.
I do not know which is preferable to include.
What do you think? Here are the headers from 10.6, for example:
https://github.com/alexey-lysiuk/macos-sdk/blob/main/MacOSX10.6.sdk/usr/include/mach/mach_port.h
https://github.com/alexey-lysiuk/macos-sdk/blob/main/MacOSX10.6.sdk/usr/include/mach/port.h

Reply:
Does htop build on your Mac if you simply include `<mach/port.h>`? If it does build, I don't think you need to include `<mach/mach_port.h>`. There is no indication that `<mach/port.h>` is a private header and that we should not include directly.

Reply:
Apparently it does. I have changed the header, as you advised, and moved the chunk below.